app/src/main/java/com/a0xffffffff/dinesum/RequestInfoActivity.java

```diff
@@ -0,0 +1,215 @@
+package com.a0xffffffff.dinesum;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Handler;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.facebook.Profile;
+import com.squareup.picasso.Picasso;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import butterknife.BindView;

```

Comment:
lol butterknife 🔪  sorry not related just thought it was funny

Reply:
🔪🔪🔪🔪🔪